enos/README.md

```diff
@@ -140,3 +140,42 @@ downloads the artifact built by the `build.yml` workflow, unzips it, and sets th
 This variant is for running the Enos scenario locally. It builds the Vault bundle
 from the current branch, placing the bundle at the `vault_bundle_path` and the
 unzipped Vault binary at the `vault_local_binary_path`.
+
+# CI Bootstrap
+In order to execute any of the scenarios in this repository, it is first necessary to bootstrap the 
+CI AWS account with the required supporting AWS resources. At this time, the only resource that is 
+required is an EC2 ssh key pair. The scenarios [bootstrap_ci](./ci/enos-scenario-bootstrap-ci.hcl) and 
+[bootstrap_workspaces](./ci/enos-scenario-ci-bootstrap-workspaces.hcl) have been created to simplify
+the process of bootstrapping the CI environment.
+
+**Supported Regions** - enos scenarios are supported in the following regions: 
+`"us-east-1", "us-east-2", "us-west-1", "us-west-2"`
+
+## Bootstrap Process
+These steps should be followed to bootstrap this repo for enos scenario execution:
+
+1. **Setup the root Workspace** - In Terraform Cloud manually create a workspace in the `hashicorp-qti` 
+   organization named `vault-ci-enos-bootstrap`. This workspace will be used as the backend for the 
+   `bootstrap_workspaces` scenario. When creating the workspace choose the execution mode `local`.
+
+
+2. Get the enos ci ssh public key from a member of the QT team.
```

Comment:
Why don't we keep the public key here in text? It's public.. there's no security risk there.

Reply:
We should tell people what the private key secret name is and where to get it. If they don't have access to that shared 1Password secret perhaps direct them to the slack channel where they can request it.